features/step_definitions/volume_snapshot.rb

```diff
@@ -7,3 +7,42 @@
     })
   step %Q/I switch to the default user/
 end
+
+# This step is to create VolumeSnapshotClass according to different paramater on differnt platforms, also it supports custom-defined parameter
```

Comment:
Typo of `paramater`.

Reply:
Thanks, updated.

Reply:
Looks like you goes to the opposite direction